src/content/pages/about.md

```diff
@@ -0,0 +1,118 @@
+---
+title: About
+url: /about
+subtitle: About
+---
+
+# About Us
+
+The EuroPython conference is organized by volunteers from the Python community
```

Comment:
The rest of the website is using UK spelling. Might be better to be consistent, i.e organization -> organisation